packages/pigeon/mock_handler_tester/test/message.dart

```diff
@@ -113,17 +143,16 @@ class NestedApi {
         details: error['details'],
       );
     } else {
-      return SearchReply.decode(replyMap['result']!);
+      // noop
```

Comment:
What's the purpose of generating a no-op else branch? There's no explanation of why it's a no-op, so it doesn't seem as an unfamiliar reader to be adding any useful information.

Reply:
Typically that is where the return statement goes.  Since this is a void returning function there is none.  I don't want to switch the structure of the function depending on wether it returns void or not so I'll have that else branch.  The `noop` was added to make it not look like a mistake, that it wasn't intentional.

Reply:
> I don't want to switch the structure of the function depending on wether it returns void or not so I'll have that else branch.

Why not? I found the generated code more confusing than if this weren't here at all. It seems like you're optimizing for ease of implementation at the cost of making the generated code (which far more people will see) more confusing.

But if you really want to keep parallel structure for some reason, why not `return;` here instead? Then the structure is more similar.

Reply:
Done. `return;` sounds fine with me.  I try to make the generated code readable but sometimes I'd say simplicity of the generator trumps output style.